core/capabilities/ccip/ccipevm/helpers_test.go

```diff
@@ -38,4 +39,45 @@ func Test_decodeExtraArgs(t *testing.T) {
 
 		require.Equal(t, gasLimit, decodedGasLimit)
 	})
+
+	t.Run("decode extra args into map evm v1", func(t *testing.T) {
+		encoded, err := d.contract.EncodeEVMExtraArgsV1(nil, message_hasher.ClientEVMExtraArgsV1{
+			GasLimit: gasLimit,
+		})
+		require.NoError(t, err)
+
+		m, err := DecodeExtraArgsToMap(encoded)
+		require.NoError(t, err)
+		require.Len(t, m, 1)
```

Comment:
Lets add `require.Equal` checks on the contents of the map.

Reply:
The check for contents is already in the below 3 lines (53-55).

Reply:
the comparison is between a map, vs fields of a struct, so require.Equals will likely not work